modules/cats-effect2/src/main/scala/pureconfig/module/catseffect/syntax/package.scala

```diff
@@ -0,0 +1,19 @@
+package pureconfig.module.catseffect
```

Comment:
Changing the package object is not enough, you also need to update all packages here and rename the folder to `catseffect2`.